home.html

```diff
@@ -0,0 +1,154 @@
+                    <img class="cardImage" src="./assets/program/icons/dice-d20-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Dice Roler
+                        </h6>
+                        <p class="cardDescription">We have virtual Dice at your disposal.</p>
+                </div>
+                </div>
+                <div class="card1">
+                    <img class="cardImage" src="./assets/program/icons/hat-wizard-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Dungeon masters
+                        </h6>
+                        <p class="cardDescription">Have a look at our registered Dungeon masters, see their rating and book with them a campaign.</p>
+                </div>
+                </div>
+                <div class="card1">
+                    <img class="cardImage" src="./assets/program/icons/comments-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Forum
+                        </h6>
+                        <p class="cardDescription">Share your thoughts and opinions.</p>
+                </div>
+                </div>
+                <div class="card1">
+                    <img class="cardImage" src="./assets/program/icons/dragon-solid.svg">
+                <div class="cardtext">
+                        <h6 class="cardtitle">
+                            Cosplayers
+                        </h6>
+                        <p class="cardDescription">See how other people show up to their campaign and post your own image.</p>
+                </div>
+                </div>
+            </div>
+        <a href="#" id="seeprogram">#See Whole Program</a>
+        <button class="JoinNPConnect" id="JoinNPConnect">Join NPConnect</button>
+            
+    </div>
+</section>
+
+<section id="dungeonMaster" class="dungeonMaster">
+    <div class="speaker-heading">
+        <h1>Famous Dungeon Masters</h1>
+        <div class="ruler"></div>
+    </div>
+    <div class="speaker-contents">
+    </div>
+</section>
+
+<section class="partners">
+    <div class="partnersContent">
+        <div class="partnersTitle">
+            <h5 id="partnersTitle1">Partner</h5>
+            <div id="ruler"></div>
+        </div>
+        <div class="partnersGroup">
+            <div class="partnersLogo">
+                <img class="partnersimage" src="./assets/partners/d-and-d-beyond.svg">
+                <img class="partnersimage" src="./assets/partners/discord.svg">
+                <img class="partnersimage" src="./assets/partners/fantasy-flight-games.svg">
+                <img class="partnersimage" src="./assets/partners/wizards-of-the-coast.svg">
+                <img class="partnersimage" src="./assets/partners/google.svg">
+            </div>
+        </div>
```

Comment:
- [ ] Kindly align the partners' section in a row and fix the color of the icons as is in the given design as per the requirement.

Your design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/abfa15ba-82c9-4901-a35b-0a95fd00ec0f)

Given design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/de314968-c1de-47a5-be89-56d064920fbd)

#### Also note the difference in sizes for mobile and desktop versions, kindly adjust your design to match the given one, this increases confidence in recruiters that you can follow any given design on a job